aws/res/bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail; IFS=$'\n\t'
+
+#
+# 기본 유틸리티들 설치
+#
+sudo yum update -y
```

Comment:
ripgrep을 이제 수동설치하는 대신 yum으로 깔 수 있는거같아요. sudo는 다 지워주세요

```suggestion
yum-config-manager --add-repo=https://copr.fedorainfracloud.org/coprs/carlwgeorge/ripgrep/repo/epel-7/carlwgeorge-ripgrep-epel-7.repo
yum update -y
```